itou/users/tests/test_management_commands.py

```diff
@@ -55,6 +57,7 @@ class DeduplicateJobSeekersManagementCommandsTest(TestCase):
     which should prevent duplication.
     """
 
+    @unittest.skipUnless(os.getenv("CI", False), "It is a management command")
```

Comment:
Coucou :) 

3 toutes petites remarques vraiment pas très graves (je m'y essaie !) :
- je trouve que `skipIf` est plus facile à comprendre pour moi, plutot que la double négation `skipUnless`, qu'en penses-tu ?
- ne peut-on pas mettre ce décorateur directement sur la classe de test, plutôt que de le mettre sur chaque test ? ce serait plus DRY ?
- le commentaire `It's a management command` me semble pas tout à fait précis: ce n'est pas parce que c'est "une management command" mais parce que celle-ci en particulier est lente et que vu son destin, on se permet de l'ignorer. Sinon on risque de se demander dans 4 mois pourquoi on ne skipperait pas toutes les management commands ^^

Reply:
J'ai effectivement mis le décorateur au niveau de la classe (merci!) et enrichi mon commentaire. Par contre, il faut bien un `skipUnless`, car ce n'est pas une double négation. Le `False` passé en paramètre est la valeur par défaut.

Reply:
En effet, je n'etais pas totalement convaincu non plus, à cause justement de cette valeur par défaut (CI  vide) - c'est juste que je croise bien moins souvent un `skipUnless` qu'un `skipIf` , mais franchement c'est très bien et compréhensible comme ça.